src/nvim/edit.c

```diff
@@ -844,6 +844,11 @@ static int insert_handle_key(InsertState *s)
     return 0;  // exit insert mode
 
 
+  case ' ':
+    if (mod_mask != 4) {
```

Comment:
@hardenedapple was there a reason not to use the symbolic form here? E.g.:

    if (mod_mask != MOD_MASK_CTRL) {